cloudformation/policies/parallelcluster-policies.yaml

```diff
@@ -637,12 +640,14 @@ Resources:
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
-          - Sid: EC2BuildImage
+          - Sid: EC2BuildAndDeleteImage
             Effect: Allow
             Action:
               - ec2:DescribeImages
               - ec2:DescribeInstanceTypeOfferings
               - ec2:DescribeInstanceTypes
+              - ec2:DeregisterImage
+              - ec2:DeleteSnapshot
```

Comment:
Why are these new actions needed?

Reply:
I got it they are not added, but just moved from another statement

Reply:
Updated the description